conductor_api/src/config.rs

```diff
@@ -314,11 +314,13 @@ impl From<AgentConfiguration> for AgentId {
 }
 
 /// A DNA is represented by a DNA file.
-/// A hash has to be provided for sanity check.
+/// A hash can optionally be provided, which could be used to validate that the DNA being installed
+/// is the DNA that was intended to be installed.
 #[derive(Deserialize, Serialize, Clone, Debug, PartialEq)]
 pub struct DnaConfiguration {
     pub id: String,
     pub file: String,
+    #[serde(default)]
```

Comment:
Should this rather be an `Option<String>`, since that would be explicitly saying what we mean here..